near/omni-types/src/lib.rs

```diff
@@ -198,18 +206,26 @@ pub enum OmniAddress {
     Arb(EvmAddress),
     Base(EvmAddress),
     Bnb(EvmAddress),
+    Btc(UTXOChainAddress),
+    Zcash(UTXOChainAddress),
 }
 
 impl OmniAddress {
     #[allow(clippy::missing_panics_doc)]
     pub fn new_zero(chain_kind: ChainKind) -> Result<Self, String> {
         match chain_kind {
-            ChainKind::Eth => Ok(Self::Eth(H160::ZERO)),
-            ChainKind::Near => Ok(Self::Near(ZERO_ACCOUNT_ID.parse().map_err(stringify)?)),
-            ChainKind::Sol => Ok(Self::Sol(SolAddress::ZERO)),
-            ChainKind::Arb => Ok(Self::Arb(H160::ZERO)),
-            ChainKind::Base => Ok(Self::Base(H160::ZERO)),
+            ChainKind::Eth => Ok(OmniAddress::Eth(H160::ZERO)),
+            ChainKind::Near => Ok(OmniAddress::Near(
+                ZERO_ACCOUNT_ID.parse().map_err(stringify)?,
+            )),
+            ChainKind::Sol => Ok(OmniAddress::Sol(SolAddress::ZERO)),
+            ChainKind::Arb => Ok(OmniAddress::Arb(H160::ZERO)),
+            ChainKind::Base => Ok(OmniAddress::Base(H160::ZERO)),
             ChainKind::Bnb => Ok(Self::Bnb(H160::ZERO)),
+            ChainKind::Btc => Ok(OmniAddress::Btc("1111111111111111111114oLvT2".to_string())),
+            ChainKind::Zcash => Ok(OmniAddress::Zcash(
+                "1111111111111111111114oLvT2".to_string(),
```

Comment:
Is it a legit address for both Zcash and Bitcoin, for mainnet and testnet?
Anyway, what is the use case of Zero address for this two chains?

Reply:
Also, we have to take into account that this is actually an burn address, not a regular zero address

Reply:
Ah no, I changed my mind, that was an important detail. We actually have quite a lot of code built on the assumption that every token has an address on each network.

When we save token information, we need to store at least some address. It can be a dummy one. Right now, instead of an address, I saved an empty string.